.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 AllCops:
-  TargetRubyVersion: 2.2
+  TargetRubyVersion: 2.5
```

Comment:
But in this way we cannot identify if there are other issue for older ruby versions.

Reply:
when it's set to 2.2 the check fails with an error about an unsupported version of Ruby, v2.5 is the lowest available version that allows Rubocop to run

Reply:
So we are not sure if we have something that can break 2.2 after this PR